CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 TiUP Changelog
 
+## [1.2.1] 2020.10.23
+
+### Improvements
+
+- Introduce a more safe way to cleanup tombstone nodes ([#858](https://github.com/pingcap/tiup/pull/858), [@lucklove](https://github.com/lucklove))
+  - When the user scale-in a TiKV server, it's data will not be deleted until he executes a display command, it's risky because there is no choise for user to confirm
```

Comment:
```suggestion
  - When an user `scale-in` a TiKV server, it's data is not deleted until the user executes a `display` command, it's risky because there is no choice for user to confirm
```